mindsdb/integrations/handlers/autogluon_handler/autogluon_handler.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict
+import pandas as pd
+from typing import Optional
+import logging
+import os
+from mindsdb.integrations.libs.base import BaseMLEngine
+from autogluon.tabular import TabularPredictor
+
+
+class AutoGluonHandler(BaseMLEngine):
+    name = 'autogluon'
+
+    # creates and trains a model inside the engine registry.
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args=None, **kwargs) -> None:
+        if 'using' in args:
+            args = args['using']
+        if 'target' in args:
+            target = args['target']
+
+        store_path = os.environ.get('MINDSDB_STORAGE_DIR') or ''
+        save_path = os.path.join(store_path, 'mindsdb-predict')  # specifies folder to store trained models
```

Comment:
You should use the model storage (e.g. `file_set()`) to store weights, because it abstracts complexity of dealing with folder and file management independent of the type of MindsDB deployment.

If you are running into issues, please post them on the community slack with the corresponding modified code snippet and we can take a closer look. If for some reason `AutoGluon.TabularPredictor` doesn't play nicely with the dir passed via `path`, another option is to train and store in-memory, then adding a secondary step post training where you serialize with `dill` and _then_ save via model storage.